debian/entry.sh

```diff
@@ -83,6 +83,7 @@ function init_non_systemd()
 		fi
 		"$CMD" "$@" &
 		pid=$!
+		wait "$pid"
```

Comment:
If you wait here for `$pid`, then what does the `fg` immediately afterwards do? Can you explain why the signal does not properly handled? (and preferably add this info in the commit message in the end).